packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.component.js

```diff
@@ -53,34 +56,32 @@ function TitleSubHeader({
 						</h1>
 					)}
 				</div>
-				{subTitle &&
-					!editMode && (
-						<small
-							className={classNames(
-								theme['tc-subheader-details-text-subtitle'],
-								'tc-subheader-details-text-subtitle',
-							)}
-						>
-							{subTitle}
-						</small>
-					)}
+				{subTitle && (
+					<small
```

Comment:
Now this subTitle is rendered even if we are in edit mode. Is this wanted ?

Reply:
Now the SubHeader component has no way to know if its EditableText is in editMode or not. I don't see other solution, but if you have suggestions I'll be glad to ear it